bokeh/core/_templates/autoload_js.js

```diff
@@ -115,16 +115,17 @@ calls it with the rendered model.
   var css_urls = {{ css_urls|json }};
 
   var inline_js = [
+    {%- for css in css_raw %}
     function(Bokeh) {
-      {%- for css in css_raw %}
-        inject_raw_css({{ css }});
-      {%- endfor %}
+      inject_raw_css({{ css }});
     },
+    {%- endfor %}
+    {%- for js in js_raw %}
     function(Bokeh) {
-      {%- for js in js_raw %}
-        {{ js|indent(6) }}
-      {% endfor -%}
-    }
+      {{ js|indent(6) }}
+    },
+    {% endfor -%}
+    function(Bokeh) {}
```

Comment:
What's the purpose of this last no-op function?

Reply:
To make sure there's no trailing comma (for IE). Otherwise I would have to use some jinja's magic loop variables.

Reply:
Comment to that effect would be helpful I think